pimd/pim_bsm.c

```diff
@@ -1282,6 +1282,13 @@ int pim_bsm_process(struct interface *ifp, struct ip *ip_hdr, uint8_t *buf,
 	bshdr = (struct bsm_hdr *)(buf + PIM_MSG_HEADER_LEN);
 	pim_inet4_dump("<bsr?>", bshdr->bsr_addr.addr, bsr_str,
 		       sizeof(bsr_str));
+	if (bshdr->hm_len > 32) {
+		zlog_warn("Garbage hashmask length; got %" PRIu8
```

Comment:
Hashmask length > 32 is not valid for IPv4. But still valid for IPv6 which we dont support right now. I would suggest removing the word - Garbage and write something to convey it is unsupported mask value for IPv4.

Reply:
Got it